harness/determined/cli/config_policies.py

```diff
@@ -0,0 +1,158 @@
+        )
+
+    wksp = api.workspace_by_name(sess, args.workspace_name)
+    body = bindings.v1PutWorkspaceConfigPoliciesRequest(
+        workloadType=workload_type, configPolicies=data, workspaceId=wksp.id
+    )
+    resp = bindings.put_PutWorkspaceConfigPolicies(
+        sess, workloadType=workload_type, workspaceId=wksp.id, body=body
+    )
+    print(util.yaml_safe_dump(resp.configPolicies, default_flow_style=False))
+    return None
+
+
+def delete_config_policies(args: argparse.Namespace) -> None:
+    sess = cli.setup_session(args)
+    workload_type = ""
+    if args.workload_type.upper() == "EXPERIMENT":
+        workload_type = "EXPERIMENT"
+    elif args.workload_type.upper() == "NTSC":
+        workload_type = "NTSC"
+    else:
+        raise cli.errors.CliError(
+            "Failed to delete config policies: Invalid workload type provided."
+            + " Valid options: 'experiment' and 'ntsc'"
+        )
+
+    wksp = api.workspace_by_name(sess, args.workspace_name)
+    bindings.delete_DeleteWorkspaceConfigPolicies(
+        sess, workloadType=workload_type, workspaceId=wksp.id
+    )
+    print(
+        f"Successfully deleted {workload_type} config policies for workspace {args.workspace_name}."
+    )
+    return None
+
+
+args_description: cli.ArgsDescription = [
+    cli.Cmd(
+        "config-policies",
+        None,
+        "manage config policies",
+        [
+            cli.Cmd(
+                "describe",
+                describe_config_policies,
+                "describe config policies",
+                [
+                    cli.Arg(
+                        "--workspace-name",
+                        type=str,
+                        required=True,  # change to false when adding --global
+                        help="config policies for the given workspace",
+                    ),
+                    cli.Arg(
+                        "--workload-type",
+                        type=str,
+                        required=True,
+                        help="config policies for the given workspace",
+                    ),
+                    cli.Group(cli.output_format_args["json"], cli.output_format_args["yaml"]),
+                ],
+                is_default=True,
+            ),
+            cli.Cmd(
+                "set",
+                set_config_policies,
+                "set config policies",
+                [
+                    cli.Arg(
+                        "--workspace-name",
+                        type=str,
+                        required=True,  # change to false when adding --global
+                        help="config policies for the given workspace",
+                    ),
+                    cli.Arg(
+                        "--workload-type",
+                        type=str,
+                        required=True,
+                        help="config policies for a given workload, options: Experiment and NTSC",
```

Comment:
```suggestion
                        help="the type (Experiment or NTSC ) of config policies to set",
```

Reply:
I don't think we necessarily have to include the command in the description for these arguments, because there is a separate help string for the command. For example, all the workload-type help strings can be the same: "the type (Experiment or NTSC ) of config policies" Wdyt?